packages/instrument-utils/src/storedInstrument/AccountInstrumentSelect/AccountInstrumentSelect.test.tsx

```diff
@@ -0,0 +1,286 @@
+    });
+
+    it('shows "use different account" label if no instrument is selected', () => {
+        render(
+            <LocaleContext.Provider value={localeContext}>
+                <Formik initialValues={initialValues} onSubmit={noop}>
+                    <Field
+                        name="instrumentId"
+                        render={(field: FieldProps<string>) => (
+                            <AccountInstrumentSelect
+                                {...field}
+                                {...defaultProps}
+                                selectedInstrumentId=""
+                            />
+                        )}
+                    />
+                </Formik>
+            </LocaleContext.Provider>,
+        );
+
+        expect(screen.getByText('Use a different account')).toBeInTheDocument();
+    });
+
+    it('shows list of instruments when clicked', () => {
+        render(
+            <LocaleContext.Provider value={localeContext}>
+                <Formik initialValues={initialValues} onSubmit={noop}>
+                    <Field
+                        name="instrumentId"
+                        render={(field: FieldProps<string>) => (
+                            <AccountInstrumentSelect {...field} {...defaultProps} />
+                        )}
+                    />
+                </Formik>
+            </LocaleContext.Provider>,
+        );
+
+        screen.getByTestId('instrument-select').click();
+
+        expect(screen.getByTestId('instrument-select-menu')).toBeInTheDocument();
+
+        expect(screen.getAllByText('test@external-id.com')).toHaveLength(2);
+    });
+
+    it('hides list of instruments by default', () => {
+        render(
+            <LocaleContext.Provider value={localeContext}>
+                <Formik initialValues={initialValues} onSubmit={noop}>
+                    <Field
+                        name="instrumentId"
+                        render={(field: FieldProps<string>) => (
+                            <AccountInstrumentSelect {...field} {...defaultProps} />
+                        )}
+                    />
+                </Formik>
+            </LocaleContext.Provider>,
+        );
+
+        expect(() => screen.getByTestId('instrument-select-menu')).toThrow();
+    });
+
+    it('notifies parent when instrument is selected', () => {
+        render(
+            <LocaleContext.Provider value={localeContext}>
+                <Formik initialValues={initialValues} onSubmit={noop}>
+                    <Field
+                        name="instrumentId"
+                        render={(field: FieldProps<string>) => (
+                            <AccountInstrumentSelect {...field} {...defaultProps} />
+                        )}
+                    />
+                </Formik>
+            </LocaleContext.Provider>,
+        );
+
+        screen.getByTestId('instrument-select').click();
+
+        screen.getAllByTestId('instrument-select-option')[1].click();
```

Comment:
Please use `userEvent` here.
https://testing-library.com/docs/ecosystem-user-event/#clickelement-eventinit-options
```suggestion
        userEvent.click(screen.getByTestId('instrument-select'));

        userEvent.click(screen.getAllByTestId('instrument-select-option')[1]);
```